docs/cli/operator.mdx

```diff
@@ -564,6 +571,367 @@ Use the options listed below to change the command's behavior.
 
 - [--visibility-uri](/cli/cmd-options#visibility-uri)
 
+## nexus
+
+Nexus commands enable managing Nexus resources.
+
+Nexus commands follow this syntax:
+
+```
+temporal operator nexus [command] [command] [command options]
+```
+
+### endpoint
+
+Endpoint commands enable managing Nexus Endpoints.
+
+Endpoint commands follow this syntax:
+
+```
+temporal operator nexus endpoint [command] [command options]
+```
+
+#### create
+
+Create a new Nexus Endpoint on the Server.
+
+An endpoint name is used in Workflow code to invoke Nexus operations.
+The endpoint target may either be a Worker, in which case `--target-namespace` and
+`--target-task-queue` must both be provided, or an external URL, in which case
+`--target-url` must be provided.
+
+This command will fail if an endpoint with the same name is already registered.
+
+```
+temporal operator nexus endpoint create \
+  --name your-endpoint \
+  --target-namespace your-namespace \
+  --target-task-queue your-task-queue \
+  --description-file DESCRIPTION.md
+```
+
+Use the following options to change the behavior of this command.
+
+- [--address](/cli/cmd-options#address)
```

Comment:
Not sure why these options are listed here without explaining what they are or what values they take